src/main/java/com/microsoft/sqlserver/jdbc/SecureStringUtil.java

```diff
@@ -104,15 +106,15 @@ private SecureStringUtil() throws SQLServerException {
      *         if error
      */
     byte[] getEncryptedBytes(char[] chars) throws SQLServerException {
-        if (chars == null)
-            return null;
-
-        byte[] iv = new byte[IV_LENGTH];
-        SecureRandom random = new SecureRandom();
-        random.nextBytes(iv);
-        GCMParameterSpec ivParamSpec = new GCMParameterSpec(TAG_LENGTH * 8, iv);
-
+        ENCRYPT_LOCK.lock();
         try {
+            if (chars == null)
+                return null;

```

Comment:
Should have been here in the first place, but please add braces for the `if` block